testing/geoclue/APKBUILD

```diff
@@ -1,34 +1,42 @@
+# Contributor: Leo <thinkabit.ukim@gmail.com>
 # Maintainer: 
 pkgname=geoclue
-pkgver=2.4.7
-pkgrel=2
+pkgver=2.5.2
+pkgrel=0
 pkgdesc="dbus geolocation service"
 url="https://www.freedesktop.org/wiki/Software/GeoClue/"
 arch="all"
-license="LGPL"
-depends=""
-depends_dev=""
-makedepends="libsoup-dev json-glib-dev modemmanager-dev avahi-dev intltool gobject-introspection-dev"
-install=""
+license="LGPL-2.1-or-later"
+makedepends="
+	meson
+	libsoup-dev
+	json-glib-dev
+	modemmanager-dev
+	avahi-dev
+	intltool
+	gobject-introspection-dev
+	libnotify-dev"
 subpackages="$pkgname-dev"
-source="https://freedesktop.org/software/geoclue/releases/2.4/geoclue-$pkgver.tar.xz"
-builddir="$srcdir/geoclue-$pkgver"
+install="$pkgname.post-install"
```

Comment:
```suggestion
install="$pkgname.pre-install"
```

Reply:
They are not required during build-time

Reply:
How will you set the permissions in the `tar` if they aren't setup when `package` runs?

Reply:
There are no geoclue-owned files in the tar, just a post-install script that creates geoclue group and user and installs an empty directory to serve as $HOME for the created geoclue user.